src/attributedlabel/unittests/NIAttributedLabelTests.m

```diff
@@ -16,18 +16,97 @@
 
 // See: http://bit.ly/hS5nNh for unit test macros.
 
+#import <Foundation/Foundation.h>
 #import <XCTest/XCTest.h>
 
 #import "NimbusAttributedLabel.h"
+#import "NIAttributedLabel+Testing.h"
+
+@interface NIAttributedLabelTestMonitor : NSObject <NIAttributedLabelDelegate>
+
+@property(nonatomic, assign) BOOL doNotRespondToLongPressCallback;
+
+@property(nonatomic, readonly) int longPressCallbackCount;
+@property(nonatomic, readonly) int actionSheetCallbackCount;
+
+@end
+
+@implementation NIAttributedLabelTestMonitor
+
+- (BOOL)respondsToSelector:(SEL)selector {
+  if (selector == @selector(attributedLabel:didLongPressTextCheckingResult:atPoint:)) {
+    return !_doNotRespondToLongPressCallback;
+  }
+
+  return [super respondsToSelector:selector];
+}
+
+- (void)attributedLabel:(NIAttributedLabel *)attributedLabel didLongPressTextCheckingResult:(NSTextCheckingResult *)result atPoint:(CGPoint)point {
+  _longPressCallbackCount++;
+}
+
+- (BOOL)attributedLabel:(NIAttributedLabel *)attributedLabel shouldPresentActionSheet:(UIActionSheet *)actionSheet withTextCheckingResult:(NSTextCheckingResult *)result atPoint:(CGPoint)point {
```

Comment:
Should this line be wrapped? It's unclear how the Github diff UI displays line limits, but I imagine this spans > 100

Reply:
Nimbus sources don't apply a column limit 📏